### PR TITLE
Various typos

### PR DESCRIPTION
**Pull request by phil-davis**
IMO these are not intentional typos for unit tests.

### CODE REVIEW

File: src/Standards/PEAR/Tests/Commenting/FunctionCommentUnitTest.inc [hunk 1]
```diff
@@ -9,7 +9,7 @@ class PHP_CodeSniffer_File
      *
      * @param int $stackPtr   The position in @ @unknown the stack of the token
      *                        that opened the scope
-     * @param int $detph    How many scope levels down we are.
```
Comment (jrfnl) [inline src/Standards/PEAR/Tests/Commenting/FunctionCommentUnitTest.inc]: This change should be reverted as this is actually testing something.

File: src/Standards/Generic/Tests/Formatting/MultipleStatementAlignmentUnitTest.js.fixed [hunk 2]
```diff
@@ -83,7 +83,7 @@ var buttonWidget     = WidgetStore.get(buttonid);
 var spinButtonid     = [this.id, '-positionFormats-spinButton'].join('');
 var spinButtonWidget = WidgetStore.get(spinButtonid);
 var position         = spinButtonWidget.getValue();
-var posForamatsList  = WidgetStore.get([self.id, '-positionFormats-list'].join(''));
+var posFormatsList  = WidgetStore.get([self.id, '-positionFormats-list'].join(''));
```
Comment (jrfnl) [inline src/Standards/Generic/Tests/Formatting/MultipleStatementAlignmentUnitTest.js.fixed]: Needs an extra space after the label. You are breaking the test now.

File: src/Standards/Squiz/Tests/Commenting/FunctionCommentUnitTest.inc [hunk 3]
```diff
@@ -9,7 +9,7 @@ class PHP_CodeSniffer_File
      *
      * @param bool   $stackPtr   The position in @ @unknown the stack of the token
      *                             that opened the scope
-     * @param int    $detph    How many scope levels down we are.
+     * @param int    $depth    How many scope levels down we are.
```
Comment (jrfnl) [inline src/Standards/Squiz/Tests/Commenting/FunctionCommentUnitTest.inc]: Same remark as before. This was testing something.

File: src/Standards/PEAR/Tests/Commenting/FunctionCommentUnitTest.inc (hunk 1 above)
Comment (phil-davis) [reply src/Standards/PEAR/Tests/Commenting/FunctionCommentUnitTest.inc]: Yeh, I tried to deduce which were "accidental" typos, and which ones were actually testing sniffs.

File: src/Standards/Generic/Tests/Formatting/MultipleStatementAlignmentUnitTest.js.fixed (hunk 2 above)
Comment (phil-davis) [reply src/Standards/Generic/Tests/Formatting/MultipleStatementAlignmentUnitTest.js.fixed]: fixed

File: src/Standards/PEAR/Tests/Commenting/FunctionCommentUnitTest.inc (hunk 1 above)
Comment (phil-davis) [reply src/Standards/PEAR/Tests/Commenting/FunctionCommentUnitTest.inc]: fixed

**Comment by gsherwood** [comment]
Thanks for these corrections.

**Pull request merged**